src/test/java/mobi/boilr/libdynticker/exchanges/BinanceExchangeTest.java

```diff
@@ -0,0 +1,45 @@
+package mobi.boilr.libdynticker.exchanges;
+
+import mobi.boilr.libdynticker.core.ExchangeTest;
+import mobi.boilr.libdynticker.core.Pair;
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class BinanceExchangeTest extends ExchangeTest {
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        testExchange = new BinanceExchange(1000);
+    }
+
+    @Test
+    public void testGetPairs() {
+        List<Pair> pairs;
+        try{
+            pairs = testExchange.getPairs();
+            Assert.assertTrue(pairs.contains(new Pair("BTC", "USDT")));
+            Assert.assertTrue(pairs.contains(new Pair("ETH", "BTC")));
```

Comment:
Missing an `assertFalse` such as:

    Assert.assertFalse(pairs.contains(new Pair("InvalidCoin", "BTC")));

Reply:
Done! :)

Reply:
Nothing here yet. Please push the new code to your repo.

Reply:
After all the new code is here. Thanks. Looking good.